.travis.yml

```
@@ -20,7 +20,7 @@ before-install:
 
 install:
   - npm install -g aglio
-  - npm install -g dredd@4.0.0
+  - npm install -g dredd@11.0.0
```

This is still old. Use latest

```suggestion
  - npm install -g dredd
```

---

Its LTS, I used the stable one .

---

Done !